apps/crn-frontend/src/network/working-groups/WorkingGroupOutput.tsx

```diff
@@ -122,7 +122,7 @@ const WorkingGroupOutput: React.FC<WorkingGroupOutputProps> = ({
           this period you will not be able to create or update research outputs
```

Comment:
We don't want to show this after the migration is over 😅 so it wasn't necessary to rebase it